.github/workflows/release-prepare.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           sed -i -E "s/SEMVER: .*/SEMVER: '${{ steps.update_version.outputs.version }}'/g" Taskfile.yml
```

Comment:
My sed-fu isn't the greatest but would those inner quotes also be optional here?

Reply:
These are there because of taskfile syntax. Desired output is this:
```yaml
SEMVER: 'v0.4.0'
```